### DISCUSSION TITLE
一直报我连不上你的网络

### DISCUSSION THREAD

**Discussion by itmryang** [zhayujie/chatgpt-on-wechat#424] [Q&A]
在群组聊天时，一直报自动回复我连不上你的网络，请问这是啥原因

**Comment by 1452534914** [comment]
我也是 你解决了吗

**Reply by Gai191** [reply]
我也是，解决了踢我一下，谢谢

**Reply by 1452534914** [reply]
我解决了，原先我用的是Visual Studio Code按照步骤走，应该是不知道哪里搞错了，换成PyCharm 继续按步骤走又好了，你可以试试，具体原因我也不太懂

**Reply by cyangshengbin** [reply]
@1452534914 我就是用的PyCharm 但是还是提示“我连接不到你的网络”。请问怎么解决啊

**Comment by BigMoonnn** [comment]
我的问题是：
[WinError 10061] 由于目标计算机积极拒绝，无法连接
